### ISSUE TITLE
The schema snapshot file is still called `datamodel.prisma`

### ISSUE THREAD

**Issue by mavilein** [prisma/prisma2#277]
Upon lift save we store a snapshot of current Prisma schema in `migrations` folder. This file is is called `datamodel.prisma` although the original file is called `schema.prisma`.

**Comment by gihrig** [comment]
Another point about`./prisma/migrations/xxxxx/datamodel.db` is `generator nexus_prisma` still contains an `output` path even though this is no longer required in `schema.prisma',

```graphql
generator nexus_prisma {
  provider = "nexus-prisma"
  output   = "node_modules/@generated/nexus-prisma"
}
```

Is this correct?

![image](https://user-images.githubusercontent.com/1481063/62078113-ae911a80-b200-11e9-941b-ea0ba127204b.png)

**Comment by timsuchanek** [comment]
Fixed by 53868b4, available in latest alpha.
